swagger/swagger.json

```diff
@@ -302,6 +297,11 @@
           "type" : "string",
           "example" : "worldpay",
           "readOnly" : true
+        },
+        "status" : {
+          "type" : "string",
+          "example" : "CREATED",
+          "readOnly" : true
```

Comment:
I'm not really sure why this change has moved this attribute in swagger, but it seems harmless enough.